runtime/tests/org.eclipse.core.tests.runtime/src/org/eclipse/core/tests/runtime/jobs/JobTest.java

```diff
@@ -1602,6 +1620,38 @@ protected IStatus run(IProgressMonitor monitor) {
 		group.done();
 	}
 
+	public void testSetProgressGroup_progressGroupIsCanceled_monitorInJobIsCanceled() throws InterruptedException {
```

Comment:
I guess the name is meant to follow some pattern like `testMethod_UseCase_ExpectedResult`, but without know that pattern (which would require to have it as default across the project) the test name's intention is hard to understand. Wouldn't something like `testSetProgressGroup_propagatesGroupCancellation` be more comprehensible and equally expressive?